source/educators/how-tos/update_rerun_course.rst

```diff
@@ -87,10 +87,9 @@ who enroll in the new course run.
 
   .. only:: Partners
```

Comment:
Actually, delete everything under `.. only:: Partners`, please.

Can you please check the other files in this pull request to see if they have anything specified as `.. only:: Partners`?

Reply:
done! I've reviewed the 27 files and no further reference for that.

Reply:
I think the ".. only::" comment still needs to be deleted, yes?

Reply:
Done